src/modules/auth/credentials/credentials.scss

```diff
@@ -168,3 +181,98 @@
 		min-height: 1px;
 	}
 }
+
+// BULK ASSIGNMENT
+.bulk-actions-wraper {
+    display: grid;
+    grid-template-areas:
+        "a a c c"
+        "a a c c"
+		"b b d d"
+		"b b d d";
+
+    grid-template-rows: 1fr 1fr 1fr 1fr;
+	grid-template-columns: 1fr 1fr 1fr 1fr;
+    column-gap: 0.25rem;
+    row-gap: 0.25rem;
+	& .card-body {
+		height: 35vh !important;
+		overflow-y: auto;
+	}
+}
+
+.credentials-suspended {
+	color: #73818f!important;
+}
+
+.credentials-list {
+    grid-area: a;
+}
+.credentials-selection {
+    grid-area: c;
+}
+
+.tenant-list {
+    grid-area: b;
+}
+.tenant-selection {
+	grid-area: d;
+}
+
+.credentials-list, .tenant-list {
+	& .data-table-col0 {
+		width: 3em !important;
+	}
+    & .data-table-thead .data-table-tr {
+		height: 0px;
+		.data-table-header-th {
+			height: 0px;
+		}
+	}
+    & .data-table-tbody {
+        background-color: var(--bg-color) ;
+        border: 0px;
+        & tbody tr {
+            border: 0px;
+        }
+    }
+
+    & .data-table-card-body .table td {
+        border: 0px;
+    }
+
+    & .data-table-tr, .data-table-tr th, .data-table-tr td {
+        height: 30px;
+    }
+    & .table-sm td, .table-sm th {
+        padding: 0;
+    }
+    tbody tr {
+        border: 0px;
+    }
+}
+
+.selected-row:hover {
+    background-color: var(--table-bg-color-hover);
```

Comment:
@petrKavulok `$table-bg-color-hover` -- you can load it from `table` styles (`@import "~asab-webui/styles/components/table.scss";`) or you can modify ASAB WebUI `index.scss` file in `styles/constants/` and import this (and maybe some other) color in there